README.md

```diff
@@ -13,143 +15,6 @@
 | `containers.*.resources.limits` / `containers.*.resources.requests` | none    | **Limits will be validated but ignored.** While the compose specification has some support for this, it is requires particular Docker versions that cannot be relied on. *This should have no impact on Workload execution*.                                                                                                                        |
 | `containers.*.livenessProbe` / `containers.*.readinessProbe`        | none    | **Probes will be validated but ignored.** The Score specification only details K8s-like HTTP probes, but the compose specification only supports direct command execution. We cannot convert between the two reliably. *This should have no impact on Workload execution*. Tracked in [#86](https://github.com/score-spec/score-compose/issues/86). |
 
-## Resource support
```

Comment:
I don't think we should remove this, this is very useful here because it is version controlled along with the readme and the release. I can go back and look at the supported resources for an older version by viewing the readme on github. It also acts as a good source of truth for the docs to be based on since the docs are not updated in the same release as this repo.